Chore/setup-improv (#1020)
web3-storage/web3.storage: packages/api/README.md

```

## Setting up a cloudflare worker

> While in most cases the [Getting Started](## Getting started) is enough to develop locally, if you want to test and preview you worker on Cloudfare infrastructure here's the instructions to setup your worker and environment.  

Reviewer: ```suggestion
While in most cases the [Getting Started](#getting-started) is enough to develop locally, if you want to test and preview you worker on Cloudfare infrastructure here's the instructions to setup your worker and environment.  
```

> If you want more in depth information please look at [Cloudfare Get Started guide](https://developers.cloudflare.com/workers/get-started/guide#1-sign-up-for-a-workers-account).  

Reviewer: ```suggestion
If you want more in depth information please look at [Cloudflare Get Started guide](https://developers.cloudflare.com/workers/get-started/guide#1-sign-up-for-a-workers-account).  
```

The following instructions assume that you did go through the Getting started section.

### 1. Sign up for a Workers account
Before you can start publishing your Workers on your own domain or a free *.workers.dev subdomain, you must sign up for a Cloudflare Workers account
### 2. Install the Workers CLI
Install `wrangler` cli on your local machine
```bash 
npm install -g @cloudflare/wrangler
```
### 3. Configure the Workers CLI
With installation complete, wrangler will need access to a Cloudflare OAuth token to manage Workers resources on your behalf.
```bash 
wrangler login
```
Open the browser, log into your account, and select Allow.

> ### 4. Configure the Workers CLI

Reviewer 2: Duplicated heading?

    account_id = "<what does the `wrangler whoami` say>"
    vars = { CLUSTER_API_URL = "https://USER-cluster-api-web3-storage.loca.lt", PG_REST_URL = "https://USER-postgres-api-web3-storage.loca.lt", ENV = "dev" }
    ```

> Copy your cloudflare account id from `wrangler whoami`.

Reviewer: ```suggestion
Copy your Cloudflare account id from `wrangler whoami`.
```

    wrangler secret put S3_SECRET_ACCESS_KEY_ID --env $(whoami) # Get from Amazon S3 (not required for dev)
    wrangler secret put S3_BUCKET_NAME --env $(whoami) # e.g web3.storage-staging-us-east-2 (not required for dev)
    wrangler secret put PG_REST_JWT --env $(whoami) # Get from database postgrest
```

> Note this might be up to date, please look to the [.env.tpl](../../.env.tpl) in the root directory for the up to date secrets required.

Reviewer 3: Might _not_ be?

## Run the code
Run `npm run build` to build the bundle
Run `npm run publish` to publish the worker under your env.

> To preview your worker using cloudfare development environment you can run

Reviewer: ```suggestion
To preview your worker using the Cloudflare development environment you can run
```

```sh
npm start:preview
```

> The script spins up the cluster, Postgres DB, Posgrest Rest interface and creates the required localtunnels to make them available to the worker.

Reviewer: ```suggestion
The script spins up the cluster, Postgres DB, Posgres Rest interface and creates the required localtunnels to make them available to the worker.
```
